### ISSUE TITLE
Sports scramble

### ISSUE THREAD

**Issue by LFC28** [LibreVR/Revive#1344]
**Which game or application is your request about?**
[Sports Scramble]

**Is your request related to a problem? Please describe.**
Game crashes after opening menu scene, around 5 seconds in.

**Describe the solution you'd like**
Would like to be able to play the game if possible.

**Describe alternatives you've considered**
Have updated to steam beta newest version and also the new revive version

**Additional context**
Would be greatly appreciated if this could be fixed, I managed to get this to load by re-installing but i couldn't replicate after closing down.(may of been a fluke)

**Comment by miraka-github** [comment]
Oculus quest+Revive  Run  Sports scramble       Error message will pop up       “  D3D buffer” 
file error         Unable to run game

**Comment by Raging-Beard** [comment]
Same, shows Oculus logo then exits to Steam VR Home.

**Comment by vredalertv** [comment]
yes I'd love to have this game work in Revive, please try with  next update!

**Comment by vredalertv** [comment]
I actually got it to work. You uninstall it then install again!

But maybe they can fix revive so you dont have to do this workaround

**Comment by miraka-github** [comment]
> I actually got it to work. You uninstall it then install again!
> 
> But maybe they can fix revive so you dont have to do this workaround

Thank you. I'll give it a try.

**Comment by Raging-Beard** [comment]
Sadly a reinstall did nothing for me :(

**Comment by Raging-Beard** [comment]
Any more help or ideas anyone please?

**Comment by thomasli54321** [comment]
I changed the compatibility for the executable. SportsScramble.exe.   One of these 3 changes seems to fix this for me, Run compatible for windows 8, run as administrator, disable full screen optimization.  Since it is working for me, will not isolate further.
